modules/virtual_network_gateway/main.tf

```diff
@@ -97,22 +97,25 @@ resource "azurerm_virtual_network_gateway" "this" {
 # https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/public_ip
 resource "azurerm_public_ip" "this" {
```

Comment:
I would place this in the top, I know it does not make more difference for TF, but it makes the code more self-documenting

Reply:
moved